extras/man/http.1

```diff
@@ -570,9 +570,10 @@ installation (only the supported ones are shown here).
 
 
 A string in the OpenSSL cipher list format.
+tls1.3 ciphers are always present regardless of your cipher list.
 
 
-See `http \fB\,--help\/\fR` for the default ciphers list on you system.
+See `http \fB\,--help\/\fR` for the default ciphers list.
```

Comment:
because it no longer use the default cipher list provided by the system.
In fact, we restore the original behavior that was removed by urllib3 (v1 to v2) where the cipher list was a constant.
we started using a consistent cipher list, using the mozilla recommended cipher suites.

Reply:
ah cool!

Reply:
in that case, I'm wondering if there's a good place generally, somewhere httpie top-level, to explain that the client cipher list available is not the system/openssl/context default, but httpie now has its own (opinionated) list that may or may not be the subset of what other cli/connections on such system are able to negotiate, which may lead to unexpected results now.

FYI the [wiki.mozilla.org/Security/Server_Side_TLS](https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29) cipher lists are meant to be server-side hardening, not client-side preference, so it may be unnecessarily restraining for a general http client w/o any prior heads up.

disclaimer: I use httpie to debug api connections where cipher + protocol (mis)match / handshake failure is common so having a different client cipher list than what the system has set (not just by default, but even intentionally by configuration in some distros)  is troublesome if it's not brought to my attention in advance.
disclaimer2: I'm involved with @mozilla TLS/SSL infosec so I might be a bit biased, but at the same time I'm well aware there are cipher suites not on the lists that are otherwise completely fine to be used if you know the context, just for the matter of simplification and automating config outputs only the "popular" enough (read: conservative and necessary) are being kept in the list, which is somewhat a dumbed-down subset compatible enough for those not able to turn the knobs a tweak the hardening tuning themselves, that can be used as-is without much understanding the impacts. So it misses CCMs, PSKs and such and that may surprise some users expecting pre-existing implicit defaults/configs on their OS level.

Reply:
> but httpie now has its own (opinionated) list that may or may not be the subset of what other cli/connections on such system are able to negotiate, which may lead to unexpected results now.

I understand your concern. There's concern on my side about your reasoning.
HTTPie did abruptly switch that behavior in a minor version due to urllib3 v2 removing "DEFAULT_CIPHERS" and there was some complaints, but not in HTTPie (aside from the breakage) as far as I have seen.

> FYI the [wiki.mozilla.org/Security/Server_Side_TLS](https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29) cipher lists are meant to be server-side hardening

I know about that. Meaning the original purpose.

My thinking / reasoning is that we need to have a cipher list that is "strong enough" and exclude de facto weak suites, and this suite, I trust. It's safe enough and permit a broad compatibility with TLS 1.2+
We live in the beginning of the "quantum cpus" era and we must be careful moving forward to protect ourselves, we've seen things no one dared to imagine with SSLv2, TLSv1 and TLSv1.1. So we must not trust too blindly the scenario "propose a wide range of ciphers" and expect the highest common cipher will be strong enough.

> list that may or may not be the subset of what other cli/connections on such system are able to negotiate

do we have a list of popular tools and the choice around ciphers?

> so having a different client cipher list than what the system has set (not just by default, but even intentionally by configuration in some distros) is troublesome if it's not brought to my attention in advance ... just for the matter of simplification and automating config outputs only the "popular" enough ...

My humble experience say the exact opposite happen very often, meaning that user does complaint with: "help! work on my mac X but not on my brand new mac Z" or "work on Linux Debian not on Fedora" with "but, same HTTPie version!" and so on. But your personal case is different.

> So it misses CCMs, PSKs and such and that may surprise some users expecting pre-existing implicit defaults/configs on their OS level.

And, maybe I am mistaken, but I am willing to bet no one actually know the extend of their "default" ciphers list (openssl lib) and how they work. So, here, actually HTTPie will offer more consistency.
Moreover, the most surprising is that users are getting blocked by popular WAF like Cloudflare if they advertise such ciphers.

If you have a way to rephrase the "help/hint/docs" for a broader audience, we accept suggestions. of course.
or maybe having a `--system-ciphers` or similar could do the trick.

regards,